lib/listeners/statful.js

```diff
@@ -3,16 +3,6 @@ var Logger = require('../logger');
 var Utils = require('../utils');
 
 /**
- * Counts the number of lines in the passed metrics lines
- *
- * @param metricLines A string containing lines of metrics
- * @returns {number}
- */
-function countMetricsInLines(metricLines) {
-    return metricLines.match(/\n/gi).length + 1;
```

Comment:
We can check the return value of the [match](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match) method here and if it is null then return 1.